doc/second-node.rst

```diff
@@ -25,127 +28,123 @@ Let's get started...
-    (If you don't have the documentation server running,
-    start it in a new terminal with ``iot doc serve`` and
-    access it at http://localhost:8001 - not needed when using
-    ``web_starter``).
+-   For reference documentation on available commands and device types,
+    see the `Command Reference <node_help/commands.rst>`_.
+    You can also start the documentation server with ``iot doc serve`` and
+    access it at http://localhost:8001 (or use ``web_starter`` which includes it).
     
-
--   Edit ``setup.cpp``
-    Either with cloudcmd edit it (with edit button or F4 key)
-    or in the command line with for example nano, micro, or tilde.
+-   Edit ``setup.cpp`` using your preferred editor (nano, micro, etc.):
 
     You should see a small explanation comment block. You should now
     completely remove it to have an empty configuration file.
 
--   In the command reference, open the button help and copy from there the
-    example configuration (including semicolon),
-    something like this: ``input(lower, D2, "depressed", "pressed");``
+-   In the command reference, look up the button configuration and copy an
+    example like: ``input(lower, D2, "depressed", "pressed");``
 
     Paste this into the setup.cpp editor.
 
 -   Change it to ``input(b1, D3, "up", "down");``
 
--   Leave the editor (press ok or by mouse the x in the upper right corner)
-    and agree to save the file.
+-   Save the file (Ctrl+S in micro, Ctrl+O then Enter in nano).
+
+-   As described in the `first node tutorial <first-node.rst>`_, 
+    flash node2 using ``deploy serial`` (or adopt it if you're using that method).
+    
+    If you need to specify a different serial port, use:
+    
+    .. code-block:: bash
 
--   Like in the `first node tutorial <first-node.rst>`_ described, 
-    adopt or flash (with deploy serial - eventually specifying the correct tty) 
-    now node2. (for adoption: attach button, activate adoption mode, count Morse blinks, 
-    use your phone to set own WiFi network credentials, finish calling
-    adopt/initialize from user menu in ``node2`` folder).
+       deploy serial /dev/ttyUSB1  # or whatever your port is
 
 
 Visually Programming the Connections
 ------------------------------------
 
--   If you don't use the Raspberry Pi image, you need to open now
-    your Node-RED server page in the browser and build the whole
-    integration flow analog to the description below.
-
--   If you use the pi image, you can head to this `Node-RED </nodered/>`_ page
-    (only works in the raspberry pi image setup - else this will lead to an undefined page).
-    Here you have to enter username (*admin*) and password (*iotempire*) again.
-
-    You should see a Node-RED flow with five nodes on the raspberry pi.
-    On a new installation, you will see an empty flow.
+-   Open your Node-RED server page in the browser (http://localhost:40080/nodered 
+    or http://gateway-ip:40080/nodered).
```

Comment:
this will now most likely run on http://localhost:40080 (the other URL is a reference to a raspberry pi gateway setup)